src/include/target/NamimnoRC_FLASH_2400_RX_ESP8285_PA.h

```diff
@@ -0,0 +1,19 @@
+#define DEVICE_NAME "Namimno 2400RX"
+
+// GPIO pin definitions
+#define GPIO_PIN_NSS            15
+#define GPIO_PIN_BUSY           5
+#define GPIO_PIN_DIO1           4
+#define GPIO_PIN_MOSI           13
+#define GPIO_PIN_MISO           12
+#define GPIO_PIN_SCK            14
+#define GPIO_PIN_RST            2
+#define GPIO_PIN_LED_RED        16
+#define GPIO_PIN_TX_ENABLE      9   // Required by diversity RX
+#if defined(USE_DIVERSITY)
+// Low = Ant1, High = Ant2, pulled high by external resistor
+#define GPIO_PIN_ANTENNA_SELECT 0
+#endif
+
+// Output Power
+#define POWER_OUTPUT_FIXED -7 // SX1280 output power for the PA (Skyworks Solutions Inc. RFX2411) is -7
```

Comment:
maybe use POWER_OUTPUT_VALUES array if we have a PA, like from Matek's RX.

Reply:
Should the values be
```cpp
#define POWER_OUTPUT_VALUES {-15, -11, -8, -5}
```
I get these from calculating the values in the data sheet of the RFX2411N and the SX1280
- RFX2411N 25dBm small signal gain on transmit
- ELRS outputs the value from -18 to 13 for the SX1280 (which is in dB increments from -18dBm)
So, given that to output 10mW (which is 10dBm) we subtract the gain on the RFX2411N and get -15 and do the same for the other values 25mW, 50mW & 100mW